src/Libraries/RevitNodesUI/Selection.cs

```diff
@@ -108,6 +148,12 @@ public override void Dispose()
             RevitServicesUpdater.Instance.ElementsDeleted -= Updater_ElementsDeleted;
             RevitServicesUpdater.Instance.ElementsModified -= Updater_ElementsModified;
             DocumentManager.Instance.CurrentUIApplication.Application.DocumentOpened -= Controller_RevitDocumentChanged;
+
+            if (null != revitDynamoModel)
+            {
+                var hwm = RevitDynamoModel.Workspaces.OfType<HomeWorkspaceModel>().ElementAt(0);
+                hwm.RunSettings.PropertyChanged -= revMod_PropertyChanged;
+            }
```

Comment:
In that case, the old workspace and its RunSettings will be disposed and so do the selection nodes in it. When RunSettings is disposed, I think the event will be automatically unsubscribed. Is that right?